src/include/stir/RadionuclideDB.h

```diff
@@ -94,16 +94,18 @@ class RadionuclideDBProcessor//: public Radionuclide
   Radionuclide radionuclide;
   
   std::string nuclide_name;
-  std::string isotope_lookup_table_str;
+  std::string radionuclide_lookup_table_str;
   
 #ifdef nlohmann_json_FOUND
 nlohmann::json radionuclide_json;
 #endif
 
+//! the following extracts the radionuclide information from the JSON file and returns a radionculide object 
   Radionuclide get_radionuclide_from_json(ImagingModality rmodality, const std::string& rname);
```

Comment:
```suggestion
  Radionuclide get_radionuclide_from_json(ImagingModality rmodality, const std::string& rname) const;
```
I think this can be done now